h2o-mapreduce-generic/src/main/java/water/hadoop/h2omapper.java

```diff
@@ -126,6 +127,19 @@ private static void repairNullArgsAndWarnIfNecessary(String[] args) {
     }
   }
   
+  static boolean makeSureIceRootExists(String iceRoot) {
+    File f = new File(iceRoot);
+    if (!f.exists()) {
+      boolean success = f.mkdirs();
+      if (!success) {
+        Log.POST(103, "mkdirs(" + f.toString() + ") failed");
```

Comment:
These Log messages should probably stay in the original location because now the method is called from different context

Reply:
moved

Reply:
actually I have decided to keep them here because we want to have it behave exactly the same, and these logs will only appear once anyways (the first time this method is called.